cmd/config.go

```diff
@@ -100,6 +101,7 @@ func writeConfig(cluster *v3.RancherKubernetesEngineConfig, configFile string, p
 }
 
 func clusterConfig(ctx *cli.Context) error {
+	logrus.Infof("Running RKE version: %v", ctx.App.Version)
```

Comment:
I think we should stick to printing version when we run a sequence that involves checking/sharing logs, I don't think running config is part of those.

Reply:
Agree with you, will change.